topics/materials-science/tutorials/muspinsim/tutorial.md

```diff
@@ -0,0 +1,932 @@
+>      - *"Interactions to simulate"*:
+>        - *"Interactions to simulate 1"*:
+>          - *"Choose interaction type"*: `hyperfine`
+>          - *"Index of nuclear coupled spin"*: `1`
+>          - *"Hyperfine coupling tensor"*: `10 0 0` `0 10 0` `0 0 10`
+>    - *"Experiment Parameters"*:
+>      - *"Experiment type"*: `custom`
+>      - *"X axis"*: `time`
+>      - *"Y axis"*: `asymmetry`
+>      - *"Average axes"*: `orientation`
+>      - *"Euler Convention"*: `ZYZ`
+>      - *"Times"*:
+>        - *"Time 1"*:
+>          - *"Time"*: `range(0, 0.1, 100)`
+>    - *"Fitting Parameters"*:
+>      - *"Fit experimental data with simulations"*: `No`
+> 2. Should generate the following output(s):
+>    - {% icon param-file %} `muspinsim input file hfine` (Output dataset)
+>
+{: .hands_on}
+
+The only changes here (other than the *"Name"*) are the addition of an electron
+spin to the system, and changing to a `hyperfine` interaction type. A
+consequence of this is that we now provide a tensor ($$3\times3$$ matrix) rather than
+a vector to define the strength and orientation of the coupling. Once again,
+the units are implicitly $$MHz$$.
+
+## MuSpinSim Simulate
+
+As all our changes are captured in the config file, this step operates exactly
+as before.
+
+> <hands-on-title>Tool details</hands-on-title>
+>
+> 1. {% tool [MuSpinSim Simulate](toolshed.g2.bx.psu.edu/repos/muon-spectroscopy-computational-project/muspinsim/muspinsim/1.1.0+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Configuration file containing simulation parameters"*: `muspinsim input file hfine` (Input dataset)
+> 2. Should generate the following output(s):
+>    - {% icon param-collection %} `Muspinsim output collection` (Output dataset)
+>    - {% icon param-file %} `muspinsim log for muspinsim input file hfine` (Output dataset)
+>
+{: .hands_on}
+
+## MuSpinSim Plot
+
+Similarly aside from labelling there are no differences with the plotting
+parameters.
+
+> <hands-on-title>Tool details</hands-on-title>
+>
+> 1. {% tool [MuSpinSim Plot](toolshed.g2.bx.psu.edu/repos/muon-spectroscopy-computational-project/muspinsim_plot/muspinsim_plot/3.5.1+galaxy0) %} with the following parameters:
+>    - *"Plot Title"*: `Hfine Example`
+>    - *"Label for x axis"*: `t`
+>    - *"Label for y axis"*: `Asymmetry`
+>    - *"Output File Type"*: `PNG`
+>    - *"Muspinsim File Series"*:
+>      - *"Muspinsim File Series 1"*:
+>        - {% icon param-file %} *"Muspinsim Experiment Data (.dat)"*: `hfine.dat`
+>        - *"Line/Point Color"*: `Black`
+>        - *"Series Type"*: `Line`
+>        - *"Line type"*: `Solid`
+>        - *"Line width"*: `1.0`
+> 2. Should generate the following output(s):
+>    - {% icon param-file %} `MuSpinSim Plot on data 4` (Output dataset)
+>
+{: .hands_on}
+
+![Resultant plot of asymmetry against time for the Zeeman Example](../../images/hfine_example.png "The asymmetry oscillates above 0 over the time of the simulation")
```

Comment:
Just noticed this text is the same as the first example.

Reply:
Great catch, thanks